api/external/applications/applications.proto

```diff
@@ -89,44 +89,44 @@ service ApplicationsService {
 
   rpc GetDisconnectedServicesConfig(GetDisconnectedServicesConfigReq) returns (PeriodicMandatoryJobConfig) {
     option (google.api.http).get = "/beta/retention/service_groups/disconnected_services/config";
-    // TODO: need new resources/policies for this
-    option (chef.automate.api.policy).resource = "service_groups";
-    option (chef.automate.api.policy).action = "list";
-    option (chef.automate.api.iam.policy).resource = "applications:serviceGroups";
-    option (chef.automate.api.iam.policy).action = "applications:serviceGroups:list";
+
+    option (chef.automate.api.policy).resource = "service_groups:scheduler:disconnected_services";
+    option (chef.automate.api.policy).action = "read";
+    option (chef.automate.api.iam.policy).resource = "retention:serviceGroups";
+    option (chef.automate.api.iam.policy).action = "retention:serviceGroups:get";
   };
 
   rpc UpdateDisconnectedServicesConfig(PeriodicMandatoryJobConfig) returns (UpdateDisconnectedServicesConfigRes) {
     option (google.api.http) = {
       post: "/beta/retention/service_groups/disconnected_services/config"
       body: "*"
     };
-    // TODO: need new resources/policies for this
-    option (chef.automate.api.policy).resource = "service_groups";
-    option (chef.automate.api.policy).action = "delete";
-    option (chef.automate.api.iam.policy).resource = "applications:serviceGroups";
-    option (chef.automate.api.iam.policy).action = "applications:serviceGroups:delete";
+
+    option (chef.automate.api.policy).resource = "service_groups:scheduler:disconnected_services";
+    option (chef.automate.api.policy).action = "configure";
+    option (chef.automate.api.iam.policy).resource = "retention:serviceGroups";
+    option (chef.automate.api.iam.policy).action = "retention:serviceGroups:update";
   };
 
   rpc GetDeleteDisconnectedServicesConfig(GetDeleteDisconnectedServicesConfigReq) returns (PeriodicJobConfig) {
     option (google.api.http).get = "/beta/retention/service_groups/delete_disconnected_services/config";
-    // TODO: need new resources/policies for this
-    option (chef.automate.api.policy).resource = "service_groups";
-    option (chef.automate.api.policy).action = "list";
-    option (chef.automate.api.iam.policy).resource = "applications:serviceGroups";
-    option (chef.automate.api.iam.policy).action = "applications:serviceGroups:list";
+
+    option (chef.automate.api.policy).resource = "service_groups:scheduler:delete_disconnected_services";
+    option (chef.automate.api.policy).action = "read";
+    option (chef.automate.api.iam.policy).resource = "retention:serviceGroups";
```

Comment:
Have we gotten product input on the namespace `retention`? To me `applications` has more meaning. From [this old spreadsheet](https://docs.google.com/spreadsheets/d/1ccaYDJdMnHBfFgmNC1n2_S1YOnMJ-OgkYd8ySbb-mS0/edit#gid=363200100), here are the other namespaces:

```

Reply:
i did just see your note in slack about job scheduler already using `retention`. so I guess it's fine 🤷‍♀